<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testCamera1Class</class>
 <widget class="QMainWindow" name="testCamera1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testCamera1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOW SRC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOW DST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>21</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>881</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_display">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>351</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshEdit">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>639</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lowLimitEdit">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>668</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setExpoTimeEdit">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>697</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPxEdit">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>726</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>640</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>669</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>698</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>937</x>
      <y>644</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>thresh:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>670</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>922</x>
      <y>694</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>724</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>return:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>610</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Information:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testCamera1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>slotStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>slotStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>showSrcBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>showDstBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>getParaBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1156</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>1189</x>
     <y>669</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>getParaBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1173</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>1192</x>
     <y>705</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>testCamera1Class</receiver>
   <slot>getParaBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1153</x>
     <y>752</y>
    </hint>
    <hint type="destinationlabel">
     <x>1197</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotStart()</slot>
  <slot>slotStop()</slot>
  <slot>slotExit()</slot>
  <slot>showSrcBtn()</slot>
  <slot>showDstBtn()</slot>
  <slot>getTotalPixBtn()</slot>
  <slot>getThreshBtn()</slot>
  <slot>getParaBtn()</slot>
 </slots>
</ui>
